larq/quantized_variable_test.py

```diff
@@ -73,7 +73,8 @@ def test_method_delegations(distribute_scope):
         if version.parse(tf.__version__) > version.parse("1.14"):
             assert x.synchronization == x.latent_variable.synchronization
         assert x.aggregation == x.latent_variable.aggregation
-        assert evaluate(x.initialized_value()) == 7
+        if version.parse(tf.__version__) < version.parse("2.13.rc0"):
```

Comment:
Should we just go with `2.13.0` here (and elsewhere in the code) instead of `rc0`?

Reply:
I don't really mind. I used `rc0` to make sure that this also works with the release candidates. But I'm happy to change it to `2.13.0` if you feel strongly about it.

Reply:
Let's remove the `rc0`, forcing people to upgrade. I think no-one should be using the release candidate anymore, right? Furthermore, elsewhere in the code you do have `version.parse("2.13")` and `version.parse("1.15")`. Or would the solution be to also do `version.parse("2.13")` here and don't include `.0` to support the release candidates and keep the code clean?

Reply:
> Or would the solution be to also do `version.parse("2.13")` here and don't include `.0` to support the release candidates and keep the code clean?

That won't be the case since `version.parse("2.13.0rc0") < version.parse("2.13") == True`

Reply:
Done in 0707ae21b87fc83cbe04debf25fdd72b328de04c